test/unit/rules/exports-valid.test.js

```diff
@@ -0,0 +1,49 @@
+const ruleModule = require('../../../src/rules/exports-valid');
+
+const {lint, ruleType} = ruleModule;
+
+describe('exports-valid Unit Tests', () => {
+  describe('a rule type value should be exported', () => {
+    test('it should equal "standard"', () => {
+      expect(ruleType).toStrictEqual('standard');
+    });
+  });
+
+  describe('when package.json has invalid node', () => {
+    const failedCases = [
+      {
+        title: 'root is `true`',
+        input: true,
+        fails: [['', 'Value should be string or object']],
+      },
+      {
+        title: 'root is a number',
+        input: 4,
+        fails: [['', 'Value should be string or object']],
+      },
+      // TODO more tests
+    ];
+    failedCases.forEach(({title, input, fails}) => {
+      // eslint-disable-next-line jest/valid-title
+      test(title, () => {
```

Comment:
Is this good Jest practice?

Reply:
I've typically seen each test explicitly called out and helpers are used to generate the data.